scripts/allowlist/README.md

```diff
@@ -0,0 +1,79 @@
+# Genesis File Content
+
+This directory contains a script (_createContent.js_) that generates some content that needs to be added to the genesis 
+file of a network that makes use of the allowlist smart contracts (_contracts/allowlist_).
+
+### Install the dependencies
+
+To install the depencies of the script run _`npm install`_ in the _scripts/allowlist_ directory. 
+
+### Input File
+
+The script reads the file _allowedAccountsAndValidators.txt_ that defines in each line the address of an allowed account and
+(optionally) the validator for that account. The account and the validator are specified using their address as a 
+hexadecimal string. If the account has a nominated validator the account and the validator hexadecimal strings need
```

Comment:
Add a comma
hexadecimal string. If the account has a nominated validator, the account and the validator hexadecimal strings need

Reply:
Done